README.md

```diff
@@ -16,7 +16,7 @@ Check the [Setup guide](/Setup.md) for information on how to run the system.
 
 ### Front-end:
 
-  * Will (soon) be available at https://arxivdigest.org.
+  * Will (soon) be available at https://arXivDigest.org.
```

Comment:
URL should be lowercase. You can also change this sentence to "Is available at ..."